scania-digital-design-system/tegel
# feat: add the possibility to flex expanded row in table

## **Describe pull-request**  
This pull request addresses issue #645. Users have reported that content within expandable rows can overflow, causing unwanted scrollbars. To resolve this, I've implemented the option to use a flex layout with overflow: hidden in expandable rows.

With this update, users can customize flex settings such as flexDirection, justifyContent, and alignItems. This enhancement provides greater control over content layout and prevents overflow issues.

I believe it's beneficial to address this now, as similar requests may arise in the future.

## **Issue Linking:**  
- **GitHub:** [Include issue link  ](https://github.com/scania-digital-design-system/tegel/issues/645)

## **How to test**  
Go to storybook, navigate to the table component and the expandable row. Adjust the controls (Flex display, Flex direction, Justify content, Align items) and make sure that they work as expected.


## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
_Include before/after screenshots for UI changes._

## **Additional context**  
The implementation is backward-compatible; existing functionality remains unaffected unless the new flex options are used.
